src/Silex/Provider/SwiftmailerServiceProvider.php

```diff
@@ -91,26 +91,28 @@ public function register(Container $app)
 
             $plugins = $app['swiftmailer.plugins'];
 
+            foreach ($plugins->keys() as $key) {
+                $dispatcher->bindEventListener($plugins[$key]);
+            }
+
+            return $dispatcher;
+        };
+
+        $app['swiftmailer.plugins'] = function ($app) {
```

Comment:
Is it necessary to lazy-load the swiftmailer.plugins container ?
If we follow your example, it would be initialized before the boot of the application.